### PR TITLE
Update mtc.py

### PR DESCRIPTION
**Pull request by assuntaciarlo**
Update functions to read and write mtc files. Add function to create mtc files.

### CODE REVIEW

File: bvbabel/mtc.py [hunk 1]
```diff
@@ -39,6 +39,24 @@ def read_mtc(filename):
         data = read_variable_length_string(f)
         header["PRT name"] = data
 
+        # Expected binary data: int (4 bytes)
+        data, = struct.unpack('<i', f.read(4))
+        header["Hemodynimic delay"] = data
```
Comment (ofgulban) [inline bvbabel/mtc.py:44]: Can you fix the type here "Hemodynimic" -> "Hemodynamic"?

File: bvbabel/mtc.py [hunk 2]
```diff
@@ -93,3 +131,55 @@ def write_mtc(filename, header, data_mtc):
...
+
+
+# =============================================================================
+def create_mtc():
+
+    """Create BrainVoyager MTC file with default values."""
+
+    header = dict()
+
+
```
Comment (ofgulban) [inline bvbabel/mtc.py:143]: Can you fix the spacing here? e.g. see `create_smp` function.

File: bvbabel/mtc.py [hunk 3]
```diff
@@ -93,3 +131,55 @@ def write_mtc(filename, header, data_mtc):
...
+    data_mtc = data_mtc.reshape((3,3))
+    data_mtc = data_mtc.astype(np.float32)
+
+
+    return header, data_mtc
+
+
+
+
+
```
Comment (ofgulban) [inline bvbabel/mtc.py:184]: Can you fix the spacings here as well? e.g. see create_smp function.

**Comment by ofgulban** [comment]
Thanks @assuntaciarlo for this contribution. It looks good, aside from a few minor issues that I have commented in your commit. If you address those lines, I am going to merge.

**Comment by assuntaciarlo** [comment]
I fixed the suggested points in the second commit. 

Regarding the reading of the ` test_data/sub-test03_cube.mtc`, with the original code the first six numbers of the mtc data appeared weird (some in the range of e-30). Adding all the header subfields as suggested here [https://support.brainvoyager.com/brainvoyager/automation-development/84-file-formats/350-users-guide-2-3-the-format-of-mtc-files](url) restored correct values and newly created mtc are also correctly read in BV (tested in version 23.2), which was not the case without all the header subfields (when the TR was not defined in the header, the link to a prt file caused some crashes while exploring the data). So I guess those fields are needed independently of the BV version at least up to 23.2.

**Comment by ofgulban** [comment]
Thanks @assuntaciarlo , looks good. Merged and closing.

**Pull request merged**